### ISSUE TITLE
Bug when tabs have similar names

### ISSUE THREAD

**Issue by BernCarney** [leram84/layer.Cake#7]
When using buttons and having them link to a tab that shares a similar name with another, it will open both. My test case was opening Plex (in a new tab) and when it would open it, it would open PlexPy (in an iFrame). Change the PlexPy tab to "Stats" fixed it so you may want to look into stricter word matching logic.

For reference, my tabs were named Plex & PlexPy when I was having the issue. Plex & Stats when it works fine.

**Comment by leram84** [comment]
yeah, its an old bug, but unfortunately the html is not auto updating. To fix it you have to either grab the new html from the repo and update manually, or remove the caret from the script like so:
`window.parent.$("li[name^='"+$(this).attr('tab')+"']").click();`
should be replaced with:
`window.parent.$("li[name='"+$(this).attr('tab')+"']").click();`

All that being said, i should warn you that some very big changes are being pushed to organizr in the next 48 hours that will make a lot of this obsolete. I mean, nothing will change if you don't want it to, but there are some big improvements coming :)

**Comment by BernCarney** [comment]
Sounds good to me! Any plans to add font customization in the future through the custom.css block or do you think this will always be handled by Organizr?

Sidenote, are the organizr updates going to be to realease or dev branch?

**Comment by leram84** [comment]
its in dev right now, but will prob be a little while longer till it all gets pushed to master. And i wouldn't try using any of the new layer cake stuff in dev atm. Cause is working on the new chat integration atm, and then we will go back and finish this theme stuff. 

As for fonts. My goal with cake proper was to provide stock color theming tool, so the only font i adjusted was the css box itself (so i could display some sweet hacker art :) lol) but I have a very massive new thing that will def play w a few fonts here and there. What exactly are you trying to do anyway? anything is possible, just let me know what you want, and i can most likely wrap it up into an addon for you.

**Comment by BernCarney** [comment]
Sounds great! Let me know if you need help with anything, I know CSS can be
a pain in the ass.

I'm looking to change most of the colored text on the homepage (Title,
Section Titles, Calendar Header, Day/Date headers)

**Comment by leram84** [comment]
hey, forgot to swing back around to this... i was obviously putting you off before cause at the time i was finishing up w blur lol. So does that work for you? or do you still prefer to stick w the default layercake themes, and still want the font pack?
